src/qt/sendcoinsdialog.cpp

```diff
@@ -337,19 +349,10 @@ void SendCoinsDialog::on_sendButton_clicked()
         if(u != model->getOptionsModel()->getDisplayUnit())
             alternativeUnits.append(BitcoinUnits::formatHtmlWithUnit(u, totalAmount));
     }
-    questionString.append(tr("Total Amount %1")
+    questionString.append(tr("<b>Total Amount</b>: <b>%1</b>")
```

Comment:
This may seem weird but I made this line's bold this way to match the bold on "Transaction fee" line where the colon is not inside bold tag since I didn't think it was good to bold the entire "Transaction fee" line. Maybe someone has a better suggestion for this.

Reply:
We should try, if possible, to keep the html/structure out of the translation strings, to make it easier for translators.
For example here it would be better to do this:
```c++
questionString.append("<b>%1</b>: <b>%2</b>".arg("Total Amount")
    .arg(...))
```
(this arises in a few other places as well)